low-level/framework/base/sspl_constants.py

```diff
@@ -227,7 +227,8 @@
 
 RESOURCE_MAP = {
     "server_type_supported": ["hw", "vm"],
-    "storage_type_supported": ["rbod", "jbod", "ebod"]
+    "storage_type_supported": ["rbod", "jbod", "ebod"],
+    "undesired_values": ["NA", "N/A", "", None]
```

Comment:
None is a valid value

Reply:
Removed it from the undesired_values list.

Reply:
Done.